packages/admin/cms-admin/src/react-split.d.ts

```diff
@@ -1,54 +0,0 @@
-declare module "react-split" {
```

Comment:
Is it okay to remove this? Don't we need the types for react-split?

Reply:
I don't think we need this. `react-split` has its own types!

<img width="418" alt="Screenshot 2024-12-05 at 07 25 15" src="https://github.com/user-attachments/assets/96a0a39f-6bbc-4f81-8b66-9d8b9f926466">